reactivity-swing/info.yml

```diff
@@ -0,0 +1,15 @@
+---
+uid: reactivity-swing
+title: Cycle reactivity swing
+prerequisites:
+  - define_reactivity
```

Comment:
(I propose that) all PR's should only be considered valid if all prerequisites are satisfied, perhaps by "empty" boundary nodes.  If this is adopted, then this PR should also define `define_reactivity`.